src/ContactComponent.php

```diff
@@ -177,6 +177,7 @@ function wf_crm_contact_access($component, $filters, $cid) {
     $utils = \Drupal::service('webform_civicrm.utils');
     // Create new contact doesn't require lookup
     $cid = (string) $cid;
+    $component['#form_key'] = $component['#form_key'] ?? $component['#webform_key'];
     list(, $c, ) = explode('_', $component['#form_key'], 3);
```

Comment:
Our #form_key-s are all over the place. Definitively a ToDo item to really get to the bottom of #form_key-s between a final beta5 release and a full release.